### ISSUE TITLE
BitBucker support

### ISSUE THREAD

**Issue by Sanchiz** [propeoplemd/cibox#272]
We need to incorporate BitBucket support for our Pull Request builder.
Here's updated BitBucket plugin https://github.com/nishio-dens/bitbucket-pullrequest-builder-plugin/pull/27 that needs re-roll. If maintainer will reject this change, we will use our custom BitBucker plugin.

**Comment by podarok** [comment]
This issue incorporates a lot of other issues like
* abstracting script for project creation and move it to non github folder
* extending CI-CI server for ability to test PR for CIBox repo for both environments
* extend documentation for github || bitbucket support

And possibly even more issues, hidden from our eyes.

And the main complexity is we are not using bitbucket for our projects <- this will slow-down all the process I guess

**Comment by Sanchiz** [comment]
@podarok Agree. I would like to suggest to put off this feature into backlog. Maybe we'll find some enthusiasts how can help, since we're not using BitBucket and it will be hard to debug functionality.

**Comment by podarok** [comment]
tagged apropriate milestone. I hope we will get some developers from community to help with a release during next year
